firebase-functions/functions/src/operations/roomOperations.ts

```diff
@@ -0,0 +1,19 @@
+import { GameType } from '../models/game-type';
+import * as admin from 'firebase-admin';
+import { collections } from '../constants/collections';
+import { Room } from '../models/room';
+
+export async function getCurrentGameType(roomId: string): Promise<GameType | undefined> {
+  const room = await admin.firestore().collection(collections.rooms).doc(roomId).get();
+  return (room.data() as Room).typeOfGame;
```

Comment:
You could call below method and avoid duplication:

`  return (await getRoom(roomId)).typeOfGame;`